docs/ngxs-labs/dispatch-decorator.md

```diff
@@ -18,8 +18,9 @@ This package simplifies dispatching process, you shouldn't care about `Store` se
 
 To install `@ngxs-labs/dispatch-decorator` run the following command:
 
-```console
-npm install @ngxs-labs/dispatch-decorator
+```bash
+npm install @ngxs-labs/dispatch-decorator --save
```

Comment:
is `--save` needed? Looks like since 2016 it's not

Reply:
https://github.com/ngxs/store/pull/527#pullrequestreview-141311222 ;)

Reply:
Ah, I see